tests/test_config.py

```diff
@@ -0,0 +1,131 @@
+#  This file is part of visiomode.
+#  Copyright (c) 2024 Olivier Delree <odelree@ed.ac.uk>
+#  Distributed under the terms of the MIT Licence.
+
+import logging
+import os
+import tempfile
+import time
+import typing
+import unittest
```

Comment:
Not really an issue but any particular reason to prefer unittest vs pytest? I've always used the latter but don't have a preference, just curious

Reply:
Same reason (and one fewer `pip install` or dependency).

Reply:
Looking at other tests that were created after this PR, they all use `pytest`. So I've switched this PR to use it as well.